neqo-transport/src/connection/tests/stream.rs

```diff
@@ -111,6 +113,121 @@ fn transfer() {
     assert!(fin3);
 }
 
+#[cfg(test)]
```

Comment:
You don't need to use `#[cfg(test)]` here.  This is in a module that already has that label.